spec/lib/msf/core/exploit/remote/remote/http/http_cookie_jar_spec.rb

```diff
@@ -0,0 +1,278 @@
+require 'spec_helper'
+require 'msf/core/exploit/remote/http/http_cookie_jar'
+require 'msf/core/exploit/remote/http/http_cookie'
```

Comment:
You shouldn't need theses `require`s 
if you run into zeitwerk issues without it let me know

Reply:
👍